lib/pbench/cli/server/user_management.py

```diff
@@ -4,15 +4,12 @@
 from pbench.cli import pass_cli_context
 from pbench.cli.server import config_setup
 from pbench.cli.server.options import common_options
-from pbench.server.database.models.users import Roles, User
+from pbench.server.database.models.users import User
 
-USER_LIST_ROW_FORMAT = "{0:15}\t{1:15}\t{2:15}\t{3:15}\t{4:20}"
-USER_LIST_HEADER_ROW = USER_LIST_ROW_FORMAT.format(
-    "Username", "First Name", "Last Name", "Registered On", "Email"
-)
+USER_LIST_ROW_FORMAT = "{0:15}\t{1:15}"
+USER_LIST_HEADER_ROW = USER_LIST_ROW_FORMAT.format("Username", "oidc id")
```

Comment:
I think a UUID is 36 characters, which won't fit well in a 15-character field (although, since you're printing it second, it overflows harmelessly...).